dashboard_server.py

```
@@ -0,0 +1,530 @@
+    return send_file(buf, mimetype='image/png')
+
+if __name__ == '__main__':
+    print("Starting LLM Judge Dashboard Server...")
+    print(f"Loaded {len(data_loader.prompt_battery)} prompts")
+    print(f"Loaded {len(data_loader.evaluation_results)} model evaluations")
+    print("Available models:", data_loader.get_model_list())
+    print("Available topics:", data_loader.get_topics())
+    print("Final results path:", FINAL_RESULTS_DIR)
+    print("\nDashboard will be available at: http://localhost:5000")
+    
+    app.run(debug=True, host='0.0.0.0', port=5000)
```

Ruff (0.12.2)

Use of `debug=True` in Flask app detected

(S201)